src/kiwipy/rmq/utils.py

```diff
@@ -107,7 +107,7 @@ def future_to_response(future):
 
 
 def ensure_coroutine(coro_or_fn):
-    if asyncio.iscoroutinefunction(coro_or_fn):
+    if inspect.iscoroutinefunction(coro_or_fn):
```

Comment:
`asyncio.iscoroutinefunction` was deprecated in 3.14,
`inspect.iscoroutinefunction` is in Python since 3.5 so safe to switch to 
https://docs.python.org/3/library/inspect.html#inspect.iscoroutinefunction